### PR TITLE
Enable Travis-CI

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
```
Comment (nex3) [inline .travis.yml]: We shouldn't run the formatter and analyzer on both dev and stable; see [the project template](https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml).

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+language: dart
```
Comment (nex3) [inline .travis.yml:1]: Add `sudo: false`.

**Review by nex3 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (kevmoo) [reply .travis.yml:1]: https://github.com/dart-lang/oss_project_templates/pull/12

File: .travis.yml (hunk 1 above)
Comment (kevmoo) [reply .travis.yml]: ack

**Comment by kevmoo** [comment]
PTAL

**Review by nex3 — APPROVED** [review]

**Pull request merged**